engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/puppet-modules/content-view-puppet-module-names.controller.js

```diff
@@ -10,16 +10,27 @@
  *   Provides functionality to the puppet modules name list.
  */
 angular.module('Bastion.content-views').controller('ContentViewPuppetModuleNamesController',
-    ['$scope', 'Nutupane', 'ContentView', function ($scope, Nutupane, ContentView) {
+    ['$scope', 'Nutupane', 'ContentView', 'CurrentOrganization', 'PuppetModule',
+    function ($scope, Nutupane, ContentView, CurrentOrganization, PuppetModule) {
 
         var nutupane = new Nutupane(
             ContentView,
             {id: $scope.$stateParams.contentViewId},
             'availablePuppetModuleNames'
         );
-
+        nutupane.masterOnly = true;
```

Comment:
Originally nutupane was designed for the 'master/child' view where you have a list, click on the list and part of the list is obscured to only leave 1 or 2 columns on the left.   This is what you see on katello's 'main' pages (products, content views, etc..).  On these pages when you click search it should close the child view and take you back to the full width list.

Slowly we started adding 'details-nutupane' tables which had no master/child interaction, it was only the main list.  In this case the nutupane would call a function and hit an error (as there was no child part of the table).

This masterOnly flag was added to indicate to nutupane that when performing different actions don't try to close details pane.

If we were implementing all this again we'd probably implement it slightly differently but it is what it is :)